### DISCUSSION TITLE
AE中导出文字动画提示 “暂不支持效果:"ADBE F建议去掉该效果， 或改用其它方式设计” 警告

### DISCUSSION THREAD

**Discussion by zhuhengyu** [Tencent/libpag#2147] [General] [answered]
![企业微信截图_17083936584985](https://github.com/Tencent/libpag/assets/30237308/87526774-917c-4b37-9466-a93612e6cfe2)

**Comment by kevingpqi123** [comment, marked as answer]
建议看下这篇文章：https://pag.art/docs/ae-bmp-guide.html
https://pag.art/docs/ae-error-guide.html
